common/src/borrow.rs

```diff
@@ -223,6 +223,8 @@ impl<M: Borrow<Market>> LinkedBorrowPosition<M> {
         let interest =
             interest_rate_part * Decimal::from(self.position.get_borrow_asset_principal());
 
+        // Assume interest will never exceed u128::MAX
+        #[allow(clippy::unwrap_used)]
```

Comment:
Wouldn't writing a `reason` be more idiomatic:
```suggestion
        #[allow(clippy::unwrap_used, reason = "Assume interest will never exceed u128::MAX")]
```
P.S. fmt might fold into two lines

Reply:
Okay, added a bunch. Thanks for this, I didn't know about this feature!